docs/cloud/how-to-manage-certificate-filters-in-temporal-cloud.md

```diff
@@ -0,0 +1,71 @@
+---
+id: how-to-manage-certificate-filters-in-temporal-cloud
+title: How to manage certificate filters in Temporal Cloud
+sidebar_label: Manage certificate filters
+description: To limit access to specific CA certificates, you can create certificate filters.
+tags:
+  - how-to
+---
+
+To limit access to specific [CA certificates](/cloud/how-to-manage-certificates-in-temporal-cloud), you can create certificate filters.
+Each filter contains values for one or more of the following fields:
+
+- commonName (CN)
+- organization (O)
+- organizationalUnit (OU)
+- subjectAlternativeName (SAN)
+
+Corresponding fields in the client certificate must match every specified value in the filter.
+In the following example, only the CN field of the certificate's subject is checked, and it must be exactly `code.example.com`.
+The other fields are not checked.
+
+```json
+AuthorizedClientCertificate {
+  CN : "code.example.com"
+}
+```
+
+In the following example, the CN field must be `stage.example.com` and the O field must be `Example Code Inc.`
+
+```json
+AuthorizedClientCertificate {
+  CN : "stage.example.com"
+  O : "Example Code Inc."
+}
+```
+
+The values for the fields are case-insensitive.
```

Comment:
Move this following section above and put examples after you have explained all the requirements?